misc/multistream-select/src/tests.rs

```diff
@@ -22,7 +22,6 @@
 
 #![cfg(test)]
```

Comment:
Not quite sure why the tests are in a separate module here. Can we move them to `tests/`?

Reply:
done

Reply:
They used private methods before, to explicitly execute the two different code paths.